src/ralph/data_importer/management/commands/initial_data.py

```diff
@@ -0,0 +1,165 @@
+        parser.add_argument(
+            '-d', '--dc-name',
+            default='dc1',
+            dest='dc_name',
+            help='Data center name.'
+        )
+        parser.add_argument(
+            '-s', '--server-room-name',
+            default='server room',
+            dest='server_room_name',
+            help='Server room name.'
+        )
+        parser.add_argument(
+            '-p', '--parent-network',
+            default='10.0.0.0/16',
+            dest='parent_network',
+            help='Parent network address.'
+        )
+        parser.add_argument(
+            '--dns1',
+            default='10.0.0.11',
+            dest='dns_1',
+            help='Primary DNS server.'
+        )
+        parser.add_argument(
+            '--dns2',
+            default='10.0.0.12',
+            dest='dns_2',
+            help='Secondary DNS server.'
+        )
+        parser.add_argument(
+            '-n', '--number-of-subnets',
+            default='3',
+            dest='number_of_subnets',
+            help='Number of /24 subnets to be generated.'
+        )
+        parser.add_argument(
+            '-r', '--region',
+            default='PL',
+            dest='region',
+            help='Geographical region (eg. your location).'
+        )
+        parser.add_argument(
+            '-c', '--configuration-path',
+            default='configuration_module/default',
+            dest='configuration_path',
+            help='Default configuration path.'
+        )
+
+    def create_users(self, region):
+        user_model = get_user_model()
+        user = get_or_create(
+            user_model, username='admin', is_staff=True, is_superuser=True
+        )
+        user.regions.add(RegionFactory(name=region))
+        user.set_password('admin')
+        user.save()
+
+    def create_configuration_path(self, configuration_path):
+        configuration_module, configuration_class = configuration_path.split(
+            '/'
+        )
+        module = ConfigurationModule.objects.create(name=configuration_module)
+        ConfigurationClass.objects.create(
+            class_name=configuration_class, module=module
+        )
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        try:
+            parent_network = ipaddress.ip_network(
+                options.get('parent_network')
+            )
+            dc_name = options.get('dc_name')
+            server_room_name = options.get('server_room_name')
+            network_address = parent_network.network_address
+            dns_1 = ipaddress.ip_address(options.get('dns_1'))
+            dns_2 = ipaddress.ip_address(options.get('dns_2'))
+            number_of_subnets = int(options.get('number_of_subnets'))
+            region = options.get('region')
+            configuration_path = options.get('configuration_path')
+        except ValueError as e:
+            raise CommandError(e)
+
+        self._validate_network(parent_network, number_of_subnets)
```

Comment:
IMO these validations should be in corresponding command classes.

Reply:
Ok.

Reply:
This validation is specific for this command. It checks sanity of the combination of specified arguments. I would rather keep it here.